.github/workflows/docker.yml

```diff
@@ -79,6 +79,19 @@ jobs:
           outputs: type=docker,dest=/tmp/${{ matrix.versions.ansible }}-${{ matrix.target }}-${{ env.PLATFORM_PAIR }}.tar
           tags: ghcr.io/${{ github.repository }}:${{ matrix.versions.ansible }}-${{ matrix.target }}-${{ env.PLATFORM_PAIR }}-${{ github.sha }}
 
+      - name: Build Docker fips images
```

Comment:
I'm also wondering if we could run the fips builds in parallel by extracting the build job in a file and reusing it, like the deploy step, WDYT? So we could build fips images in parallel of the regular ones.
Just a random thought, feel free to ignore.

Reply:
it's a good idea, but not sure if it's worth it

Reply:
Sure, totally up to you